web/cypress/e2e/TestRunDetail/Outputs.spec.ts

```diff
@@ -0,0 +1,102 @@
+describe('Outputs', () => {
+  it('should create outputs from attributes/scratch', () => {
+    cy.createTest();
+
+    cy.intercept({method: 'GET', url: '/api/tests/**/run/**/select?query=*'}).as('getSelect');
+
+    // Open output flow from the Trace view (attributes)
+    cy.selectRunDetailMode(2);
+    cy.get('[data-cy=trace-node-database]', {timeout: 25000}).first().click({force: true});
+    cy.get('[data-cy=attributes-search-container] input').type('db.name');
+    cy.get('[data-cy=attribute-row-db-name] .ant-dropdown-trigger').click();
+    cy.contains('Create output').click();
+
+    // Save output
+    cy.wait('@getSelect');
+    cy.get('[data-cy=output-save-button]').click();
+    cy.get('[data-cy=output-pending-tag]').should('have.length', 1);
+
+    // Add new output from scratch
+    cy.get('[data-cy=output-add-button]').click();
+    cy.get('form#testOutput').within(() => {
+      cy.get('#testOutput_name').type('status_code');
+      cy.get('[data-cy=selector-editor] [contenteditable=true]').type('span[tracetest.span.type = "http"]:first');
```

Comment:
I like this, what does `within` does?

Reply:
Hey, it scopes the following `cy` commands to the parent element. Very useful if you want to narrow your selectors.